cpp/src/barretenberg/honk/proof_system/work_queue.hpp

```diff
@@ -0,0 +1,143 @@
+#pragma once
```

Comment:
It seems like you're deciding (rightly IMO) that the honk work queue is going to not share code with the plonk work queue. In that case, will you move the work queue from `proof_system` to `plonk`? Otherwise we create tech debt.

Reply:
good call, done

Reply:
Great, thanks.